src/services/db/RepoService.ts

```diff
@@ -113,9 +113,36 @@ export default class RepoService extends GitHubService {
   }
 
   async create(
-    sessionData: any,
-    { content, fileName, directoryName, isMedia = false }: any
-  ): Promise<any> {
+    sessionData: UserWithSiteSessionData,
+    {
+      content,
+      fileName,
+      directoryName,
+      isMedia = false,
+    }: {
+      content: string
+      fileName: string
+      directoryName: string
+      isMedia?: boolean
+    }
+  ): Promise<{ sha: string }> {
```

Comment:
Same here, this should be `GitCommitResult`.

Reply:
The result here needs to be `{ sha: string }` in order to match the result of the `githubservice.create`!

Reply:
Oh gosh 😭

Reply:
I think we can leave it as it is right now, but I think we can potentially refactor in the future so that they have the same return types